### DISCUSSION TITLE
how to localizating (and X more) in laravel (v9)

### DISCUSSION THREAD

**Discussion by aliakbarebrahimy** [laravel/framework#43514] [Q&A]
- Laravel Version: 9.1.9
- PHP Version: 8.0.2

### Description: [how to localizating (and X more) in laravel (v9)](https://stackoverflow.com/questions/73203324/how-to-localizating-and-x-more-in-laravel-v9)

**Comment by henzeb** [comment]
I've seen that earlier. Currently it's hardcoded. but feel free to create a PR for that. https://github.com/laravel/framework/blob/9.x/src/Illuminate/Validation/ValidationException.php#L96

**Reply by aliakbarebrahimy** [reply]
thanks , but summarize (line 86) method in this file is not localizable , 
for example : 
'The given data was invalid.'
' (and {$additional} more {$pluralized})'

**Reply by aliakbarebrahimy** [reply]
Can you share this problem with the Laravel developer community?

**Reply by henzeb** [reply]
That's what I said. It's hard coded. I am not working for Laravel, and you could create either a bug report or a pull request if you want to change that.
